CHANGES_NEXT_RELEASE

```diff
@@ -8,4 +8,5 @@ Do not allow 'id' and 'type' as attribute names
 Change parameter for POST action from 'URL' to 'url'
 Fix error in VR for updateAction (#55)
 Fix validation rule name (#44, #15)
+Fix negative and zero chek intervals for NSR (#53)
```

Comment:
chek -> check

In additon I'd include the expansion of NSR (not sure right now what that acronym means :)